pai-management/bootstrap/hadoop-node-manager/hadoop-node-manager-configuration/nodemanager-generate-script.sh

```diff
@@ -65,3 +68,7 @@ sed  -i "s/{cpu_vcores}/${cpu_vcores}/g" $HADOOP_CONF_DIR/yarn-site.xml
 
 # Backup hadoop configuration for job (Spark) use
 cp $HADOOP_CONF_DIR/* /hadoop-configuration-for-jobs/
+
+# Copy docker .config file for yarn-container
+cp -a /root/.docker /docker-config/
```

Comment:
What purpose to add this?

Reply:
I thought it is for private registry support.
But after checking the code, the private job container is authenticated in yarnContainerScript, this volume is redundant and should be removed.

Another question, what's the use of mount it to node manager?

Reply:
Can we remove this line and corresponding volume？

Reply:
I'm not sure whether other features depend on this configmap, in my test, removing this volume from nm yaml seems no difference